src/components/Modal.tsx

```diff
@@ -15,7 +15,7 @@ const LegoModal: React.FC<Props> = ({
   onSubmit,
   disabled,
 }) => (
-  <Modal ariaHideApp={false} isOpen={true} className="_legoEditor_modal_modal">
```

Comment:
Not a fan of replacing classnames with inline styling - any reason for it?

Reply:
This classname didn't do anything :grimacing:.
react-modals API for styling is just inline I'm pretty sure